pkg/node/node.go

```diff
@@ -273,6 +273,11 @@ func NewBee(interrupt chan struct{}, addr string, publicKey *ecdsa.PublicKey, si
 			return nil, fmt.Errorf("batchstore: %w", err)
 		}
 	}
+	post, err := postage.NewService(stateStore, batchStore, chainID)
```

Comment:
Why is this moved? This should be moved back to where it was originally. The Set needs to happen after this is done.